docker/Makefile

```diff
@@ -45,6 +45,7 @@ PDO_USER_UID ?= $(shell id -u)
 PDO_GROUP_UID ?= $(shell id -g)
 
 DOCKER_DIR = ${PDO_SOURCE_ROOT}/docker
+DOCKER_ENCLAVE_SIGNING_PEM = $(DOCKER_DIR)/xfer/services/keys/sgx/enclave_code_sign.pem
```

Comment:
you're copying this key into the image right? xfer is specifically for transferring runtime files into the running containers. i think this is the wrong place to put the key.

Reply:
Yes. That took some time, to get it right, and for the following reasons:
* PDO_ENCLAVE_CODE_SIGN_PEM can be already set to some key on the host
* copying the key into the image requires the key in a relative path w.r.t. the context, and the perfect location is the xfer folder, so it's copied there
* the copy operation in the docker file does not known the final destination of the key, because that's in the environment.sh within the "tools" (copied in the image immediately before); so the key is temporarily copied in /tmp
* then a bash script in the container loads the environment.sh to grab the final location of the key and moves it there
* similarly, after build, a bash script grabs the current/final location of the key to delete it